auth/whoami.go

```diff
@@ -0,0 +1,58 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ninech/nctl/api"
+)
+
+type WhoAmICmd struct {
+	APIURL     string `help:"The URL of the Nine API" default:"https://nineapis.ch" env:"NCTL_API_URL" name:"api-url"`
+	IssuerURL  string `help:"Issuer URL is the OIDC issuer URL of the API." default:"https://auth.nine.ch/auth/realms/pub"`
+	ClientID   string `help:"Client ID is the OIDC client ID of the API." default:"nineapis.ch-f178254"`
+	ExecPlugin bool   `help:"Automatically run exec plugin after writing the kubeconfig." hidden:"" default:"true"`
+}
+
+func (s *WhoAmICmd) Run(ctx context.Context, client *api.Client) error {
+	cfg, err := ReadConfig(client.KubeconfigPath, client.KubeconfigContext)
+	if err != nil {
+		if IsConfigNotFoundError(err) {
+			return ReloginNeeded(err)
+		}
+		return err
+	}
+
+	userInfo, err := api.GetUserInfoFromToken(client.Token)
+	if err != nil {
+		return err
+	}
+
+	printUserInfo(userInfo, cfg)
+
+	return nil
+}
+
+func printUserInfo(userInfo *api.UserInfo, cfg *Config) {
+	fmt.Printf("You are currenlty logged in the with the following account: %q\n", userInfo.User)
+
+	fmt.Printf("Your current organization: %q\n", cfg.Organization)
+
+	printAvailableOrgsString(cfg.Organization, userInfo.Orgs)
+}
+
+func printAvailableOrgsString(currentorg string, orgs []string) {
+	fmt.Print("Active\tOrganization\n")
```

Comment:
Last nits about this:

* this line should not be printed if there are no orgs, e.g. for a service account
* another line break before would be good to better separate this table from the rest of the output